test/visualTest.ts

```diff
@@ -229,6 +232,171 @@ describe("Sunburst", () => {
                     done();
                 }, 2, DefaultWaitForRender);
         });
+
+        it("data label font size should be correct", (done: DoneFn) => {
+            dataView = defaultDataViewBuilder.getDataView(
+                [
+                    defaultDataViewBuilder.RegionsDataSet,
+                    defaultDataViewBuilder.CountriesDataSet
+                ]);
+            const fontSize: number = 22;
+            const expectedFontSize: string = "22px";
+            dataView.metadata.objects = {
+                group: { labelFontSize: fontSize }
+            };
+            visualBuilder.updateRenderTimeout(
+                dataView,
+                () => {
+                    const firstClickPoint: HTMLElement = visualBuilder.slices[visualBuilder.slices.length - 1];
+                    const secondClickPoint: HTMLElement = visualBuilder.mainElement[0];
+                    d3Click(firstClickPoint, 5, 5);
+                    setTimeout(
+                        () => {
+                            d3Click(secondClickPoint, 1, 1);
+                            setTimeout(
+                                () => {
+                                    expect((<HTMLElement>visualBuilder.element.querySelector(SliceLabelSelector)).style.fontSize).toBe(expectedFontSize);
+                                    done();
+                                },
+                                DefaultWaitForRender);
+                        },
+                        DefaultWaitForRender);
+                },
+                2,
+                DefaultWaitForRender);
+        });
+
+        it("data label font weight should be correct", (done: DoneFn) => {
+            dataView = defaultDataViewBuilder.getDataView(
```

Comment:
use beforeEach() for repetitive code in tests

Reply:
Done: [7a120ff](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/7a120ff3160317fefdf7313f99f253101efb7f43)